features/cellular/framework/API/CellularDevice.h

```diff
@@ -222,12 +222,14 @@ class CellularDevice {
     /** Stop the current operation. Operations: set_device_ready, set_sim_ready, register_to_network, attach_to_network
      *
      */
+    MBED_DEPRECATED_SINCE("mbed-os-5.15", "Use CellularDevice::shutdown() instead.")
```

Comment:
@AriParkkila If the other one is not yet decided how to go about it, can you keep this one at least and we progress?

Reply:
@0xc0170 updated